pom.xml

```diff
@@ -1,9 +1,10 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.10</version><!-- which version of Jenkins is this plugin built against? -->
+        <version>2.36</version><!-- Built against the Jenkins version specified in properties as 'jenkins.version' -->
```

Comment:
Updated Parent POM [changelog]( https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md)

Comment noted here: https://github.com/jenkinsci/office-365-connector-plugin/blob/d27e25fca4c3a623f6f7fcbd8bd9b5a0d8007152/pom.xml#L27

Reply:
@casz Change in the version from 2.10 to 2.36 is failing in Netbeans. Am getting the below error:
--- maven-enforcer-plugin:3.0.0-M1:display-info (display-info) @ Office-365-Connector ---
Maven Version: 3.0.5
JDK Version: 1.8.0_144 normalized as: 1.8.0-144
OS Info: Arch: amd64 Family: windows Name: windows 10 Version: 10.0

--- maven-enforcer-plugin:3.0.0-M1:enforce (display-info) @ Office-365-Connector ---
Ignoring requireUpperBoundDeps in com.google.guava:guava
Ignoring requireUpperBoundDeps in com.google.code.findbugs:jsr305
Rule 0: org.apache.maven.plugins.enforcer.RequireMavenVersion failed with message:
3.1.0 required by frontend-maven-plugin at least.

My maven version is 3.3.9... Am not sure why netbeans is not taking the higher version. My IDE is up to date. Any Idea how I can fix this?

Reply:
It's the built in I guess... That is the reason it is failing. When I build it externally, everything works as the version of maven installed in 3.3.9... How can I change it in netbeans?

Reply:
Tools > Options > Java > Maven > Maven Home...
![image](https://user-images.githubusercontent.com/1661688/31615166-09022db6-b289-11e7-963a-0e33064767fe.png)

Reply:
Thanks a ton... :)